docs/tutorial_tx_service_initiate_sign.md

```diff
@@ -23,7 +23,7 @@ Gnosis is currently running the backend for these Ethereum networks:
 ## Proposing transactions
 
 This tutorial will teach you how to propose transactions to the transaction service, there are 2 reasons you would want to do that:
-- The transactions appear on the UIs, so you can craft custom transactions that can be signed and send by any of the owners in any of the UIs. Please note: Transactions submitted as a non-owner will not show up on the official Safe interfaces unless you previously added a delegate. There is a separate [tutorial on how to add a delegate to your Safe](tutorial_tx_service_set_delegate).
+- The transactions appear on the UIs, so you can craft custom transactions that can be signed and send by any of the owners in any of the UIs. Please note: Transactions submitted as a non-owner will not show up on the official Safe interfaces unless you previously added a delegate. There is a separate [tutorial on how to add a delegate to your Safe](https://docs.gnosis.io/safe/docs/tutorial_tx_service_set_delegate).
```

Comment:
It should be similar to how it is done here https://github.com/gnosis/safe-docs/blob/develop/docs/tutorial_tx_service_set_delegate.md .... so basically you were missing the `.md` at the end

Reply:
I think it should be alright now @rmeissner, I changed both links